stellarphot/photometry/tests/fake_image.py

```diff
@@ -24,10 +25,46 @@ class FakeImage:
         the seed will be randomly generated.
     """
 
-    def __init__(self, noise_dev=1.0, seed=None):
-        self.image_shape = [400, 500]
+    def __init__(self, noise_dev=1.0, seed=None, fwhm=None, n_repeats=1):
+        self.image_shape = np.array([400, 500])
         data_file = get_pkg_data_filename("data/test_sources.csv")
         self._sources = Table.read(data_file)
+        self.input_fwhm = fwhm
+        if fwhm is not None:
+            # Set the FWHM of the sources to the specified value
+            self._sources["x_stddev"] = gaussian_fwhm_to_sigma * fwhm
+            self._sources["y_stddev"] = gaussian_fwhm_to_sigma * fwhm
+        if n_repeats > 1:
+            amplitude_scales = np.logspace(
+                start=np.log10(0.1), stop=np.log10(60.0), num=n_repeats**2
+            )
+            # Reshape to n_repeats x n_repeats
+            amplitude_scales = amplitude_scales.reshape((n_repeats, n_repeats))
+            extra_sources = []
+            for i in range(n_repeats):
```

Comment:
This nested pair of loops seems to suggest you apprend `n_repeats` SQUARED sources to `extra_sources`.  That doesn't seem to make sense given the parameter  name.  Can you clarify what this nested loop is supposed to do.

Reply:
It is supposed to produced `n__repeats**2` copies. I'll come up with a better name when I document the arguments.

Reply:
Renamed the variable to `n_repeats_per_side`